draft-ietf-scitt-architecture.md

```diff
@@ -470,15 +471,15 @@ The Transparency Service provides a history of Statements, which may be made by
 
 Transparency is implemented by providing a consistent, append-only, cryptographically verifiable, publicly available record of entries.
 A SCITT instance is referred to as a Transparency Service.
-Implementations of Transparency Services may protect their Append-only Log using a combination of trusted hardware, replication and consensus protocols, and cryptographic evidence.
-A Receipt is an offline, universally-verifiable proof that an entry is registered in the Append-only Log.
+Implementations of Transparency Services may protect their Append-only Log and Verifiable Data Structure using a combination of trusted hardware, consensus protocols, and cryptographic evidence.
+A Receipt is an offline, universally-verifiable proof that an entry is registered in the Verifiable Data Structure.
```

Comment:
```suggestion
A Receipt is a signature over one or more Verifiable Data Structure Proofs that an entry is registered in the Verifiable Data Structure. It is universally verifiable without online access to the TS.
```

Split for clarity?

Reply:
I particularly like "It is universally verifiable without online access to the TS"

Reply:
That has a stronger privacy posture than CT I think

Reply:
I'm committing the suggestion, we can review in totality.